ios/AppLovinMAX.m

```diff
@@ -655,6 +655,12 @@ - (instancetype)init
 RCT_EXPORT_METHOD(loadInterstitial:(NSString *)adUnitIdentifier)
 {
     MAInterstitialAd *interstitial = [self retrieveInterstitialForAdUnitIdentifier: adUnitIdentifier];
+    if ( !interstitial )
+    {
+        [self sendReactNativeEventForAdLoadFailed: @"OnInterstitialLoadFailedEvent" forAdUnitIdentifier: adUnitIdentifier withError: nil];
+        return;
+    }
```

Comment:
Is this possible? And this PR is for adview logic, we should keep it to such unless this is a related change.

Reply:
Removed.  I thought that it'd be nice to be consistent with the Java side.